contracts/SpokePool.sol

```diff
@@ -61,7 +61,7 @@ abstract contract SpokePool is SpokePoolInterface, Testable, Lockable, MultiCall
         // 256x256 leaves per root.
         mapping(uint256 => uint256) claimedBitmap;
     }
-    RootBundle[] public rootBundles;
+    RootBundle[] public rootBundles; // todo: think if this has to be an array or if mapping is better
```

Comment:
Remaining TODO @chrismaree @mrice32 

I think we can leave this here unless we decide to hash the `RootBundle` data and map it to a bool?

Reply:
ye, I think the array structure is ok. the bool option could work as well but it's not clearly better. thoughts @mrice32 ?

Reply:
No I don't think the bool option works well if the data structure has multiple elements in it. I think one option that might be cheaper would be a slow relay mapping to bool and a separate relayerRefundRoot mapping to a bool + claimedBitMap mapping.

However, that would be a bit more complicated.